S2_week4作業

peihsuan1225 opened the pull request:
周姵萱
http://52.9.113.1:8000/
http://52.9.113.1:8000/attraction/10

來交作業，麻煩老師了!

peihsuan1225 committed:
# create member table and test data, add if table exsit or not function

peihsuan1225 committed:
# user api*3, signin & signup popup window, create member table
- user api*3: signup, signin get token, get current member information
- signin & signup popup window: display, fetch api, css modify, interact
- create member table: create table and test data

cwpeng reviewed (changes requested):
- We should check user sign-in status by calling User API, not just check localStorage in the front-end. (how about I put any text in localStorage through developer tools?)
- Format of request data should align with User API specs.

peihsuan1225 committed:
# request modify, signin status check rule  modify
- request modify: change request to class basemodel, change request title
- signin status check rule modify: fetch api to check status

peihsuan1225 commented:
周姵萱
http://52.9.113.1:8000/
http://52.9.113.1:8000/attraction/10

修改好了，麻煩老師!

cwpeng reviewed (approved):
OK. Done.
